library/build.gradle

```diff
@@ -67,6 +67,8 @@ dependencies {
 
     implementation "com.google.code.gson:gson:$gsonVersion"
 
+    implementation "org.brotli:dec:$brotliVersion"
```

Comment:
Open question:

Have you tried if this works if you use `BrotliInterceptor` from  `okhttp-brotli`:
https://search.maven.org/artifact/com.squareup.okhttp3/okhttp-brotli/4.2.0/jar
https://square.github.io/okhttp/4.x/okhttp-brotli/okhttp3.brotli/-brotli-interceptor/.

I suppose if one of our consumer is using Brotli + OkHTTP they will most likely use it. 
Therefore, I'm unsure if `Headers.containsBrotli: Boolean` would even work correctly (based on the order of the interceptors).

Reply:
I think it should work. Gzip works in the same way. But it's worth checking.

Reply:
Tested with `BrotliInterceptor` being before `ChuckerInterceptor` and the opposite order - everything works fine and response with headers is in place.